armada_jupyter/podspec.py

```diff
@@ -30,4 +33,11 @@ def create_podspec_object(podspec_dict):
                     for key, value in container["resources"]["requests"].items()
                 }
 
+    # check that length of containers is exactly 1
```

Comment:
We may want to relax this at some point.  I don't fully understand how multi node jobs get scheduled in Armada but MPI and other jobs could use others containers.  

We should file a ticket to at least document this as a limitation right now.

Reply:
I did this as a design choice to be honest. With jobs with multiple containers, it becomes hard to figure out what the URL will be.

Reply:
Yea I guess I'm thinking about support for something like Dask or Ray.  But we can tackle that if there is a request.

Reply:
Sounds good to me :)))